entrypoint.sh

```diff
@@ -67,7 +67,7 @@ while http_status=$(curl -o /dev/null -w '%{http_code}' -sL "${NOTICE_URL}"); [
 done
 
 echo "$(date): ${http_status}"
-MESSAGE="Spot Termination${CLUSTER_INFO}: ${NODE_NAME}, Instance: ${INSTANCE_ID}, Instance Type: ${INSTANCE_TYPE}, AZ: ${AZ}"
+MESSAGE="Spot Termination ${CLUSTER_INFO}: ${NODE_NAME}, Instance: ${INSTANCE_ID}, Instance Type: ${INSTANCE_TYPE}, AZ: ${AZ}"
```

Comment:
The lack of the space here is deliberate - see line 52 for how it is set. If it is not set, then you do not want a space...

Reply:
Thanks. Fixing!